app/models/mixins/vim_connect_mixin.rb

```diff
@@ -49,11 +40,7 @@ module ClassMethods
     def raw_connect(options)
       options[:pass] = MiqPassword.try_decrypt(options[:pass])
       validate_connection do
-        if options[:fault_tolerant]
-          MiqFaultTolerantVim.new(options)
-        else
-          MiqVim.new(options[:ip], options[:user], options[:pass])
-        end
+        MiqFaultTolerantVim.new(options)
```

Comment:
:+1: for getting rid of the option, but want @jrafanie to approve as well.

Reply:
Get rid of it, I don't see anything setting it.

We'll find out if something needed it.

```
02:53:26 ~/Code/manageiq (master) (2.4.2) + git grep fault_tolerant
app/models/mixins/vim_connect_mixin.rb:    options[:fault_tolerant] = true unless options.key?(:fault_tolerant)
app/models/mixins/vim_connect_mixin.rb:    options[:use_broker] = (self.class.respond_to?(:use_vim_broker?) ? self.class.use_vim_broker? : ManageIQ::Providers::Vmware::InfraManager.use_vim_broker?) if options[:fault_tolerant] && !options.key?(:use_broker)
app/models/mixins/vim_connect_mixin.rb:    require 'VMwareWebService/miq_fault_tolerant_vim'
app/models/mixins/vim_connect_mixin.rb:    if options[:fault_tolerant]
app/models/mixins/vim_connect_mixin.rb:        if options[:fault_tolerant]
tools/vim_collect_inventory.rb:  require 'VMwareWebService/miq_fault_tolerant_vim'
tools/vim_collect_perf_history.rb:  require 'VMwareWebService/miq_fault_tolerant_vim
```